presto-main/src/main/java/com/facebook/presto/sql/planner/LocalExecutionPlanner.java

```diff
@@ -1779,6 +1806,7 @@ else if (context.getDriverInstanceCount().isPresent()) {
 
                 DriverFactory driverFactory = new DriverFactory(subContext.isInputDriver(), false, operatorFactories, subContext.getDriverInstanceCount());
                 context.addDriverFactory(driverFactory);
+                context.setGroupIdSymbol(subContext.getGroupIdSymbol());
```

Comment:
@sopel39  I'm really not sure about this line. Are you familiar with this part of the code? What kind of query would produce multiple inputs to a local exchange?